### PR TITLE
fix(report): scroll and resize issue

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by jbrundage** [comment]
@GordonSmith @dcamper 
I have no idea how I could have missed these issues in testing...

File: report/src/index.ts [hunk 1]
```diff
@@ -40,29 +40,39 @@ export class ReportTabs extends DockPanel {
...
+                                } else {
+                                    this.addWidget(r.report, r.result.Name, "tab-after", resultReports[i - 1].report);
+                                }
+                            });
+                            super.render(w => {
+                                if (callback) {
+                                    callback(this as any);
+                                }
+                            });
+                        });
```
Comment (GordonSmith) [inline report/src/index.ts]: This looks wrong, I suspect something more like:
```
        if(!this._prevFetch){
            this._prevFetch = this._wu.fetchResults()                .then(results => {
                    Promise.all(results.filter(isProfileResult).map(result => {
                            return result.fetchRows().then(rows => {
                                return {
                                    result,
                                    report: new Report(rows)
                                }
                            });
                        })).then((resultReports: any) => {
                            resultReports.forEach((r: any,i) => {
                                if (i === 0) {
                                    this.addWidget(r.report, r.result.Name);
                                } else {
                                    this.addWidget(r.report, r.result.Name, "tab-after", resultReports[i - 1].report);
                                }
                            });
                        });
                });
       }
        this._prevFetch.then(()=>{
                            super.render(w => {
                                if (callback) {
                                    callback(this as any);
                                }
                            });
        });
        }
```

File: report/src/index.ts [hunk 2]
```diff
@@ -40,29 +40,39 @@ export class ReportTabs extends DockPanel {
...
                 });
+        }
+        super.render(w => {
+            if (callback) {
+                callback(this as any);
+            }
```
Comment (GordonSmith) [inline report/src/index.ts]: This looks wrong, I suspect something more like:
```
        if(!this._prevFetch){
            this._prevFetch = this._wu.fetchResults()                .then(results => {
                    Promise.all(results.filter(isProfileResult).map(result => {
                            return result.fetchRows().then(rows => {
                                return {
                                    result,
                                    report: new Report(rows)
                                }
                            });
                        })).then((resultReports: any) => {
                            resultReports.forEach((r: any,i) => {
                                if (i === 0) {
                                    this.addWidget(r.report, r.result.Name);
                                } else {
                                    this.addWidget(r.report, r.result.Name, "tab-after", resultReports[i - 1].report);
                                }
                            });
                        });
                });
       }
        this._prevFetch.then(()=>{
                            super.render(w => {
                                if (callback) {
                                    callback(this as any);
                                }
                            });
        });
        }
```

Comment (jbrundage) [reply report/src/index.ts]: I think this would prevent `super.render` from being called on subsequent calls to render?

Comment (jbrundage) [reply report/src/index.ts]: I've made the above change and it does call super.render on subsequent calls to render

Comment (GordonSmith) [reply report/src/index.ts]: I don't think so.

**Comment by dcamper** [comment]
@GordonSmith Thoughts on the latest version of the code?

**Pull request merged**